test/common.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/fields"
+)
+
+const (
+	AnnotationKey   = "wetigerswoop"
+	AnnotationValue = "hurra"
```

Comment:
nit: We try to consistently use references to _Arrested Development_ (Mostly from S01E01) in example values. 
That saud, I don't see why we want these constants defined as global constants; this should be a per-test thing.

If we define common values, then usually it's more like (`clarify-go/test: true`) or something like that. I.e. a namespaces annotation that can be used to avoid conflict with other stuff.